elk/training/eigen_reporter.py

```diff
@@ -105,6 +105,18 @@ def contrastive_xcov(self) -> Tensor:
     def intercluster_cov(self) -> Tensor:
         return self.intercluster_cov_M2 / self.n
 
+    @property
+    def confidence(self) -> Tensor:
+        return self.weight.mT @ self.intercluster_cov @ self.weight
+
+    @property
+    def invariance(self) -> Tensor:
+        return self.weight.mT @ self.intracluster_cov @ self.weight
```

Comment:
Sorry can you add a negative sign to this? Right now it's _higher_ when there's _more_ variation across prompts (so less invariance).

The idea is that, for each of these metrics, it should be "higher is better" roughly.